Update standard library DIU Beta 4 release notes (#1357)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta4-release-note.md

#### New Features

##### MySQL Package
- Introduced failover and retries support

> - Added noAccessToProcedureBodies options

Reviewer: ```suggestion
- Added `noAccessToProcedureBodies` options
```

##### Log Package

> - Introduced `setOutputFile` function to write log output to a file.

Reviewer: ```suggestion
- Introduced the `setOutputFile` function to write the log output to a file
```

#### Improvements

##### SQL Package

> - Improved queryRow() function to support union return types.

Reviewer: ```suggestion
- Improved the `queryRow()` function to support union return types
```

> - Improved parameterised query to support escaped backtick as insertions.

Reviewer: ```suggestion
- Improved the parameterized query to support the escaped backtick as insertions
```

##### Log Package

> - Added `error:StackFrame[]` as a key-value pair type.

Reviewer: ```suggestion
- Added `error:StackFrame[]` as a key-value pair type
```
